specs/src/specs/ante_handler.md

```diff
@@ -0,0 +1,23 @@
+# AnteHandler
+
+Celestia makes use of a Cosmos SDK [AnteHandler](https://docs.cosmos.network/v0.46/modules/auth/03_antehandlers.html#antehandlers) in order to reject decodable sdk.Txs that do not meet certain criteria. The ante handler is defined in [app/ante/ante.go](https://github.com/celestiaorg/celestia-app/blob/7f97788a64af7fe0fce00959753d6dd81663e98f/app/ante/ante.go). It chains together several decorators to ensure the following criteria are met:
```

Comment:
Can you add (somewhere appropriate) that it's used in four areas: `CheckTx` when a transaction goes into the mempoool, `PrepareProposal` for when building a block, `ProcessProposal`, for validating blocks and `DeliverTx`, for executing the transaction

Reply:
Good idea! Resolved in 223a80f1c93c5e3e44b3ecfbfcb01262356c6625